src/index.ts

```diff
@@ -57,7 +57,7 @@
  * A linear least square solver using the Conjugate Gradient method.
  *
  * * Solve sparse linear least square problems.
- * * Solve sparse constrained linear least square problems using a lagrange method.
+ * * Solve sparse constrained linear least square problems using a Lagrange method.
```

Comment:
Eigen below is capitalized. I don't see any reason not to treat Lagrange with similar respect.